packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -0,0 +1,782 @@
+    terms: { 
+      view: 'usersWithOptedInToDialogueFacilitation',
+      limit: 10, 
+      sort: { karma: -1 } 
+    },
+    fragmentName: 'UsersOptedInToDialogueFacilitation',
+    collectionName: 'Users'  
+  });
+
+  let targetUserIds = [];
+  if (userDialogueChecks) {
+    targetUserIds = userDialogueChecks.map(check => check.targetUserId);
+  }
+
+  async function createDialogue(title: string, participants: string[]) {
+    const createResult = await createPost({
+      data: {
+        title,
+        draft: true,
+        collabEditorDialogue: true,
+        coauthorStatuses: participants.map(userId => ({userId, confirmed: true, requested: false})),
+        shareWithUsers: participants,
+        sharingSettings: {
+          anyoneWithLinkCan: "none",
+          explicitlySharedUsersCan: "edit",
+        },
+        contents: {
+          originalContents: {
+            type: "ckEditorMarkup",
+            data: ""
+          }
+        } as AnyBecauseHard
+      },
+    });
+    if (createResult?.data?.createPost?.data) {
+      const post = createResult?.data?.createPost?.data;
+      if (post) {
+        const postId = post._id;
+        const postEditUrl = `/editPost?postId=${postId}`;
+        history.push(postEditUrl);
+      }
+    }
+  }
+
+  if (!currentUser) return <p>You have to be logged in to view this page</p>
+  if (loading) return <Loading />
+  if (error || !userDialogueChecks || userDialogueChecks.length > 1000) return <p>Error </p>; // if the user has clicked that much stuff things might break...... 
+  if (userDialogueChecks?.length > 1000) {
+    console.log("Warning: userDialogueChecks.length > 1000, seems user has checked more than a thousand boxes? how is that even possible? let a dev know and we'll fix it...")
+  }
+
+  const handleOptInToRevealDialogueChecks = async (event: React.ChangeEvent<HTMLInputElement>) => {
+    setOptIn(event.target.checked);
+    void updateCurrentUser({revealChecksToAdmins: event.target.checked})
+    captureEvent("optInToRevealDialogueChecks", {optIn: event.target.checked})
+    
+    if (event.target.checked) {
+       // ping the slack webhook to inform team of opt-in. YOLO:ing and putting this on the client. Seems fine. 
+      const webhookURL = "https://hooks.slack.com/triggers/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" // TODO dev mode link for now, update before pushing to prod "https://hooks.slack.com/triggers/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";
+      const userDetailString = currentUser?.displayName + " / " + currentUser?.slug
+      const data = { user: userDetailString };
+      void pingSlackWebhook(webhookURL, data)
+    }
+  };
+
+  const prompt = "Opt-in to LessWrong team viewing your checks, to help proactively suggest and facilitate dialogues" 
+
+  let isMobile = false;
+  if (typeof window !== 'undefined') {
+    isMobile = window.innerWidth <= 768; // Adjust this value as needed
+  }
```

Comment:
We have an `isMobile` helper function [defined elsewhere](https://github.com/ForumMagnum/ForumMagnum/pull/8117#discussion_r1382116264) - check `lib/utils/isMobile.ts`